ffi/diplomat/src/properties_maps.rs

```diff
@@ -21,9 +21,7 @@ pub mod ffi {
     #[diplomat::rust_link(icu::properties::maps::CodePointMapDataBorrowed, Struct)]
     pub struct ICU4XCodePointMapData8(maps::CodePointMapData<u8>);
 
-    fn convert_8<P: TrieValue + 'static>(
-        data: maps::CodePointMapData<P>,
-    ) -> Box<ICU4XCodePointMapData8> {
+    fn convert_8<P: TrieValue>(data: maps::CodePointMapData<P>) -> Box<ICU4XCodePointMapData8> {
```

Comment:
this is unrelated to `DataMarker`, but it's a `'static` bound that is already part of `TrieValue`.